src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace APIViewWeb.Repositories
+{
+    public class DevopsArtifactRepository
+    {
+        static readonly HttpClient devopsClient = new();
```

Comment:
nit: naming

Reply:
I would look in the direction of something like Polly our Azure.Core pipeline to get retries timeouts etc.

Reply:
I have filed an issue to make change in a separate PR if that is fine. https://github.com/Azure/azure-sdk-tools/issues/2013